src/amo/tracking.js

```diff
@@ -180,19 +183,32 @@ export class Tracking {
         _getFID(sendWebVitalStats);
         _getLCP(sendWebVitalStats);
       }
+
+      // GA4 setup
```

Comment:
So we have the setup and the `_ga4` method, but shouldn't we use it ? I don't see any changes to `setPage`/`pageView`/`sendEvent` ? And indeed I don't see any network activity for GA4 when browsing around, after the initial google tag manager script load.

Reply:
The `config` is supposed to set up "automatic events", which include page views. With GA4 you don't have to send any page view events, nor is `setPage` relevant anymore. We are also using this new `_ga4` method in `sendWebVitalStats` to send an event with the Web Vitals, so we are using it at least once. 

For this patch my goal was to simply set it up and prove that it's working. I wasn't planning on migrating all of the events to GA4 just yet. The call to `_ga4` in `sendWebVitalStats` should be enough to prove that we can send a custom event to GA4. I still don't know why it's not working from localhost, but I am asking around, and hoping we might have better luck testing from dev.

Reply:
So currently we are not supposed to see anything, and you believe there is some kind of check client-side that's preventing the data to be sent ? I see `window.dataLayer` getting filled up, but never flushed out to the network.

Reply:
I was manually inspecting it in the devtools console (before/after browsing around)

Reply:
I will try to do a bit of digging to see if I can figure out why it's not getting flushed. I would like to get this onto dev soon, though, in case we find it's easier to test and verify there.